public/static/docs/sidebar.json

```diff
@@ -253,6 +253,10 @@
             "label": "metrics modify",
             "slug": "modify"
           },
+          {
+            "label": "metrics diff",
```

Comment:
need to update index and other metric files to include a link to this new section?

Reply:
Done too.